mycore-acl/src/main/resources/components/acl/config/mycore.properties

```diff
@@ -39,6 +39,8 @@ MCR.RestApi.Draft.MCRAccessKey=true
 ##############################################################################
 # Access Key
 ##############################################################################
-# e.g. {read, writedb}
-MCR.ACL.AccessKey.Session.AllowedPermissionTypes=read,
+# MCR.ACL.AccessKey.Strategy.AllowedObjectTypes=object,derivate
+MCR.ACL.AccessKey.Strategy.AllowedObjectTypes=object
```

Comment:
please disable properties per default and change "object" to "mods" or "document"